src/taxRatesByZip/README.txt

```diff
@@ -1 +0,0 @@
-This folder stores tax rates by ZIP code.
```

Comment:
I think we need to leave this in, its going to get regenerated by SDK generator.